### ISSUE TITLE
Add unit dropdown to font size

### ISSUE THREAD

**Issue by simurai** [atom/settings-view#735]
Currently the Font Size is unit-less and `px` are used.

![screen shot 2016-02-18 at 9 19 37 am](https://cloud.githubusercontent.com/assets/378023/13129566/116e7bda-d621-11e5-9c4b-492216ab260e.png)

## Enhancement

Add a "Font Unit" dropdown that lets you chose between different units. According to @damieng, Win users prefer `pt`. And it will render better than `px`. We could also add a few more options.. maybe:

- `px`
- `pt`
- `%` used in browsers
- `em` font size will change depending on the font size of the UI theme.
- `vw` + `vh` font size will change depending on the window width/height, which might be nice for certain use cases.
- More?

**Comment by abe33** [comment]
Maybe `rem`? It might be less confusing than `em` as it won't be affected by parents of the editor (which can happen when a text editor is composed by another package).

**Comment by abe33** [comment]
On a side note, it would be great to have the ability to specify a unit hint for any numeric field. The bootstrap's `input-group-addon` could be use for that purpose.

**Comment by simurai** [comment]
> Maybe `rem`?

Yes, that would map straight to a UI theme's `@font-size` variable: https://github.com/atom/atom/blob/master/static/workspace-view.less#L8

> ability to specify a unit hint

That, would look nice yeah.

![screen shot 2016-02-23 at 12 46 01 pm](https://cloud.githubusercontent.com/assets/378023/13241119/9fe52cde-da2b-11e5-82be-af2be820031f.png)

With "hint", do you mean it would be "read only" and not an actual drop-down to change the unit. Maybe both.
